programs/ziti-prox-c/proxy.c

```diff
@@ -415,6 +415,30 @@ const char *my_configs[] = {
         "all", NULL
 };
 
+
```

Comment:
It think this has to be :axe: or provided real implementations

Reply:
Will remove.

Reply:
Removed.